src/modules/esl-footnotes/core/esl-note.ts

```diff
@@ -83,14 +83,21 @@ export class ESLNote extends ESLBaseElement {
     return this.allowFootnotes ? `${this._index}` : this.standaloneLabel;
   }
 
+  /** Note settings of ignoring by footnotes */
+  protected get ignoreFootnotes(): string {
+    return this._ignoreFootnotes;
+  }
+  protected set ignoreFootnotes(value: string) {
```

Comment:
It looks a little bit strange for just setter because it can ignore value, while in fact, it's valid.
Can I suggest a little bit different composition here:
1) Have just getter to receive actual ignore state:
```
  public get ignoreQuery() {
     const ignore = this.getClosestRelatedAttr('ignore') || this.ignore || 'not all'; // Falback to simplify checks
     return ignore;
     // Or even
     return ESLMediaQuery.for(ignore)
  }
```
2) Memoize it just in purpose to return a currently applied state to the outside world and have an old state to release resources.
```
  @memoize
  public get ignoreQuery() { ... }
```
3) Setup update flow in the following way:
```
   public updateIgnoreQuery() { // or refreshIgnoreQuery
      this.ignoreQuery.removeListener(this._onBPChange);
      memoize.clear(this, ignoreQuery);
      // Note that subscription is lightweight (as it is ignored) for constant queries
      this.ignoreQuery.addListener(this._onBPChange); 
   }
```

Reply:
Thanks for the suggested way to fix the crooked design.